spec/30-processing-model.md

```diff
@@ -11,7 +11,7 @@ This processing model describes the behavior of a vendor that modifies and forwa
 If no traceparent header is received:
 
 1. The vendor checks an incoming request for a `traceparent` and a `tracestate` header.
-2. If _no_ `traceparent` header is received, the vendor creates a new `trace-id` and `parent-id` that represents the current request.
+2. Because the `traceparent` header is not received, the vendor creates a new `trace-id` and `parent-id` that represents the current request.
```

Comment:
I think item 2 and item 3 are two branches in a processing model. If not recieved - do this, if recieved - do that

Reply:
This entire section is titled _No traceparent Received_. This means that the _if_ in item 2 is superfluous and asserts a condition which is always true in this section and thus is unnecessary.